output/schema/schema.json

```diff
@@ -5314,6 +5314,53 @@
         }
       ]
     },
+    {
+      "availability": {
+        "serverless": {
+          "featureFlag": "esql_views",
+          "stability": "experimental",
+          "visibility": "feature_flag"
+        },
+        "stack": {
+          "featureFlag": "esql_views",
+          "since": "9.3.0",
+          "stability": "experimental",
+          "visibility": "feature_flag"
+        }
+      },
+      "description": "Delete an ES|QL view.\n\nDeletes a stored ES|QL view.",
+      "docId": "esql-delete-view",
+      "docUrl": "",
+      "name": "esql.delete_view",
+      "privileges": {
+        "cluster": [
+          "monitor_esql"
```

Comment:
This will be replaced by the real privileges once they exist: `delete_view, manage_view`